subsys/bluetooth/rpc/Kconfig

```diff
@@ -107,6 +107,12 @@ config BT_RPC_GATT_BUFFER_SIZE
 
 endif # BT_RPC_HOST
 
+config BT_RPC_INTERNAL_TEST
```

Comment:
Why we need additional config to add source file with `bt_addr_le_is_bonded` function ?